javascript/ql/src/semmle/javascript/frameworks/React.qll

```diff
@@ -54,10 +54,20 @@ abstract class ReactComponent extends ASTNode {
   /**
    * Gets the `this` node in an instance method of this component.
    */
-  DataFlow::SourceNode getAThisAccess() {
+  DataFlow::SourceNode getAThisNode() {
```

Comment:
Can this predicate still have more than one result? If not, `getThisNode` (without the "A") would perhaps be a better name, and fits better with the doc comment.

Also, presumably there are some uses of the now-deprecated old version below that need updating?

Reply:
`git grep getAThisAccess` doesn't find any uses, so I guess it's not used in our own libraries.

It returns one `ThisNode` per instance method (not including constructor, but maybe it should). We could canonicalize `this` nodes per class as well, but that's a bit more controversial so I didn't do it in this PR.

Reply:
OK, thanks for the clarification.